arcflow/utils/bulk_import.py

```diff
@@ -128,8 +172,122 @@ def csv_bulk_import(csv_directory=None, load_type='ao', only_validate='false'):
                 }),
             }
         ).json()
+        file_import_report["results_status"] = import_job.get("status")
+        file_import_report["results_id"] = import_job.get("id")
+        file_import_report["results_uri"] = import_job.get("uri")
+        file_import_report["results_warnings"] = import_job.get("warnings")
+
+        bulk_import_report.append(file_import_report)
         print(json.dumps(import_job, indent=4))
+    return bulk_import_report
```

Comment:
Since `retrieve_job_output` modifies the output of `csv_bulk_import`, I recommend to put `retrieve_job_output` inside `csv_bulk_import` to maintain logical cohesion, and add an `asnake_client` parameter to `retrieve_job_output` (this also eliminates the need to instantiate an additional `ASnakeClient`).

Part 3 of 4
```suggestion
        bulk_import_report.append(file_import_report)                
        print(json.dumps(import_job, indent=4))
        
    if save_output_files:
        retrieve_job_output(csv_directory, bulk_import_report, client)
        
    return bulk_import_report
```